resources/js/Components/TableComponent.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+  <table
+    ref="TableComponent"
+    :class="[
+      'wikit',
+      'wikit-Table',
+      `wikit-Table--linear-${breakpoint}`
```

Comment:
@guergana @hasanakg renaming those lines breaks the whole table, but I'm not actually sure why, we don't assign any explicit styles to these selectors. any ideas?

Reply:
can you explain it a bit more @chukarave ?

Reply:
You can use ```['table-component', `table-component--linear${breakpoint}`]```, right now this doesn't use the style from `TableComponent.vue`

Reply:
nice! I think you got it